code/game/objects/items.dm

```diff
@@ -48,6 +48,9 @@ GLOBAL_DATUM_INIT(welding_sparks, /mutable_appearance, mutable_appearance('icons
 //	causeerrorheresoifixthis
 	var/obj/item/master = null
 
+	/// Is this an objective item?
+	var/objective_item = FALSE
+
```

Comment:
not a fan of where this is scoped. This will be used on max maybe 10 items. Just use signals

Reply:
Traits*